openstudiocore/src/energyplus/ForwardTranslator.cpp

```diff
@@ -313,6 +313,15 @@ Workspace ForwardTranslator::translateModelPrivate( model::Model & model, bool f
     }
   }
 
+  // remove orphan Generator:PVWatts
```

Comment:
should we do the same thing for inverters?

Reply:
Does this not cover it? https://github.com/NREL/OpenStudio/blob/pvwatts/openstudiocore/src/energyplus/ForwardTranslator.cpp#L353-L358